src/pe_source/dnstwistscript.py

```diff
@@ -0,0 +1,261 @@
+    )
+    if test == 1:
+        return dnstwist_result
+    finalorglist = dnstwist_result + []
+    for dom in dnstwist_result:
+        if ("tld-swap" not in dom["fuzzer"]) and ("original" not in dom["fuzzer"]):
+            LOGGER.info("Running again on %s", dom["domain"])
+            secondlist = dnstwist.run(
+                registered=True,
+                tld=pathtoDict,
+                format="json",
+                threads=8,
+                domain=dom["domain"],
+            )
+            finalorglist += secondlist
+    return finalorglist
+
+
+def run_dnstwist(orgs_list):
+    """Run DNStwist on certain domains and upload findings to database."""
+    PE_conn = connect()
+    source_uid = getDataSource(PE_conn, "DNSTwist")[0]
+
+    """ Get P&E Orgs """
+    orgs = query_orgs_rev()
+    failures = []
+    for org_index, org_row in orgs.iterrows():
+        pe_org_uid = org_row["organizations_uid"]
+        org_name = org_row["name"]
+        pe_org_id = org_row["cyhy_db_name"]
+
+        # Only run on orgs in the org list
+        if pe_org_id in orgs_list or orgs_list == "all":
+
+            LOGGER.info("Running DNSTwist on %s", pe_org_id)
+
+            """Collect DNSTwist data from Crossfeed"""
+            try:
+                # Get root domains
+                rd_df = org_root_domains(PE_conn, pe_org_uid)
+                domain_list = []
+                perm_list = []
+                for i, row in rd_df.iterrows():
+                    root_domain = row["root_domain"]
+                    if root_domain == "Null_Root":
+                        continue
+                    LOGGER.info("Running on %s", row["root_domain"])
+
+                    finalorglist = execute_dnstwist(root_domain)
+
+                    # Get subdomain uid
+                    sub_domain = root_domain
+                    try:
+                        sub_domain_uid = getSubdomain(PE_conn, sub_domain)[0]
+                    except Exception:
+                        # TODO: Create custom exceptions.
+                        # Issue 265: https://github.com/cisagov/pe-reports/issues/265
+                        LOGGER.info("Unable to get sub domain uid", "warning")
+                        # Add and then get it
+                        addSubdomain(PE_conn, sub_domain, pe_org_uid)
+                        sub_domain_uid = getSubdomain(PE_conn, sub_domain)[0]
+
+                    for dom in finalorglist:
+                        domain_dict, perm_list = checkBlocklist(
+                            dom, sub_domain_uid, source_uid, pe_org_uid, perm_list
+                        )
+                        if domain_dict is not None:
+                            domain_list.append(domain_dict)
+            except Exception:
+                # TODO: Create custom exceptions.
+                # Issue 265: https://github.com/cisagov/pe-reports/issues/265
+                LOGGER.info("Failed selecting DNSTwist data.", "Warning")
```

Comment:
Consistent capitalization:
```suggestion
                LOGGER.info("Failed selecting dnstwist data.", "Warning")
```